app/admin/tenants/onboarding.py

```diff
@@ -211,15 +211,13 @@ async def onboard_issuer(
         )
 
         await acapy_ledger.accept_taa_if_required(issuer_controller)
-        # TODO: This doesn't need endorsement as of 0.7.5-rc0 onward - bug in 0.7.4
-        # Change this in future versions
+        # NOTE: Still needs endorsement in 0.7.5 release
+        # Otherwise did has no associated services.
```

Comment:
What is exactly going wrong here? Thought it would be fixed in 0.7.5. If we can find the root cause we can open an issue in ACA-Py

Reply:
I'm still trying to find that out. The error is something along the lines of "cannot to connect via public did with no associated services".